dts/bindings/serial/nxp,kinetis-lpuart.yaml

```diff
@@ -15,6 +15,26 @@ properties:
       disconnected, and transmitter output is internally connected to
       the receiver input.
 
+  single-wire:
```

Comment:
Should we have a pinctrl state for this mode? Or will the normal pinctrl state suffice since the RX and TX lines are connected internally? I guess my core question here is do we need to setup pin muxing for the RX and TX pin in single wire mode, or only the TX pin?

Reply:
I guess if pinctrl state is used where only one is used then the other pin could be used for something else

Reply:
Indeed the TX pin becomes a half-duplex pin. The other pin can be anything else.

Reply:
@danieldegrasse  are you suggesting that the driver support another pinctrl mode, is it really needed or can user just configure the default pinctrl to only mux one pin on their board?

Reply:
I guess it isn't needed, no. My reasoning what that (like the flow control state for this driver) it allows a board like the EVK to define pin control states for all the modes the LPUART driver may run in, so the user could just set the `single-wire` property and have the correct pin control settings applied on their EVK.

Reply:
> I guess it isn't needed, no. My reasoning what that (like the flow control state for this driver) it allows a board like the EVK to define pin control states for all the modes the LPUART driver may run in, so the user could just set the `single-wire` property and have the correct pin control settings applied on their EVK.

I think it would be better to handle with an overlay than adding code to the driver